src/app/shared/search/search-result.component.ts

```diff
@@ -151,4 +159,8 @@ export class SearchResultComponent implements OnChanges {
     }
     return isAscending ? SortOrder.ASCENDING : isDescending ? SortOrder.DESCENDING : SortOrder.UNSORTED;
   }
+
+  removeDraftId(resultViewItem: SearchResultViewItem): string {
```

Comment:
this seems to be used only by the html. Shall we move this to a  pipe?

Reply:
perhaps another solution would be to change the search view to not display the case_id, but a new field `displayed_case_id` of the SearchResultViewItem
for a draft displayed_case_id = 'Draft', for a case it's it's case_id
this way we won't have to mutate the case_id, which could be source of issues